CheckThePossibilityBuiltThreeAngle.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+public class CheckThePossibilityBuiltThreeAngle {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the lengths of threeangle");
+        int Side1 = sc.nextInt();
+        int Side2 = sc.nextInt();
+        int Side3 = sc.nextInt();
+        if (Side1 <= 0 || Side2 <= 0 || Side3 <= 0) {
+            System.exit(1);
```

Comment:
And what user can understand by just seeing that the program has been terminated without any explanation?
Put a meaningful message before terminating the program.

Reply:
Sure. Will be done.